src/store/Workspaces/cheWorkspaces/index.ts

```diff
@@ -113,8 +116,9 @@ type WorkspaceStatusMessageHandler = (message: api.che.workspace.event.Workspace
 type EnvironmentOutputMessageHandler = (message: api.che.workspace.event.RuntimeLogEvent) => void;
 const subscribedWorkspaceStatusCallbacks = new Map<string, WorkspaceStatusMessageHandler>();
 const subscribedEnvironmentOutputCallbacks = new Map<string, EnvironmentOutputMessageHandler>();
+const onStatusChangeCallbacks = new Map<string, (status: string) => Promise<void>>();
```

Comment:
I think callbacks should not be obliged to return promises

```suggestion
const onStatusChangeCallbacks = new Map<string, (status: string) => void>();
```

Reply:
I have fixed it.